Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +979,44 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandBootstrapOption {
+    private let apply : ShorthandApply
```

Comment:
More a stylistic note, but in general I'd recommend struct members should be `var` unless there's a compelling invariant reason to use `let`.

Reply:
```suggestion
    private let apply: ShorthandApply
```